drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/client.rb

```diff
@@ -417,6 +475,13 @@ def self.sections
           title: 'Overlapping Homeless Service After Move-in in PH',
           description: 'Client\'s receiving more than two overlapping homeless nights are included.',
           required_for: 'Adults',
+          detail_columns: [
+            :destination_client_id,
+            :first_name,
+            :last_name,
+            :reporting_age,
+            :overlapping_post_move_in,
```

Comment:
Not sure this is right based on the name, but it matches the description on line 45...
```suggestion
            :overlapping_pre_move_in,
```